pep-0514.txt

```diff
@@ -13,8 +13,8 @@ Abstract
 ========
 
 This PEP defines a schema for the Python registry key to allow third-party
-installers to register their installation, and to allow applications to detect
-and correctly display all Python environments on a user's machine. No
+installers to register their installation, and to allow tools and applications
+to detect and correctly display all Python environments on a user's machine. No
 implementation changes to Python are proposed with this PEP.
 
```

Comment:
Cool, thanks. Maybe add those links as footnotes in the PEP?

In that "add an environment" dialog, how much is required? I ask because I'd argue that if any of the fields are mandatory, then the corresponding data item in the PEP should probably be mandatory (not "unknown").

I'm not sure about your >90% figure, we get a lot of bug reports for pip and virtualenv from Windows users. I can't imagine those typically come from IDE users. Like everything, I suspect it depends on what particular interest area you deal with, though.